internal/services/compute/shared_image_version_resource_test.go

```diff
@@ -120,6 +120,157 @@ func TestAccSharedImageVersion_blobURI(t *testing.T) {
 	})
 }
 
+func TestAccSharedImageVersion_uefiSettingsSignatureTemplateNames(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_shared_image_version", "test")
+	r := SharedImageVersionResource{}
+
+	signatureTemplateNames := `["MicrosoftUefiCertificateAuthorityTemplate"]`
+	updatedSignatureTemplateNames := `["MicrosoftUefiCertificateAuthorityTemplate", "MicrosoftWindowsTemplate"]`
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			// need to create a vm and then reference it in the image creation
+			Config:  r.setup(data),
+			Destroy: false,
+			Check: acceptance.ComposeTestCheckFunc(
+				data.CheckWithClientForResource(ImageResource{}.virtualMachineExists, "azurerm_virtual_machine.testsource"),
+				data.CheckWithClientForResource(ImageResource{}.generalizeVirtualMachine(data), "azurerm_virtual_machine.testsource"),
+			),
+		},
+		{
+			// First create a shared image version with initial signature Template Name
+			Config: r.imageVersionUefiSettingsTemplates(data, signatureTemplateNames),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		// The security profile and hence the uefi_settings are not returned by the API
+		// so we need to ignore them
+		data.ImportStep(
```

Comment:
Have you tried directly use `data.ImportStep("uefi_settings")` ?

Reply:
@ms-zhenhua Using `data.ImportStep("uefi_settings")` will work but my impression was, naming every diff would be "more correct". This is e.g. also done here: https://github.com/hashicorp/terraform-provider-azurerm/blob/6755758d0ef15481cf4c7a32790370597c18166d/internal/services/mssqlmanagedinstance/mssql_managed_instance_resource_test.go#L1151-L1160

I am also fine to make it shorter. Your decision.